src/business/dal/api/api.connection.ts

```diff
@@ -0,0 +1,48 @@
+import axios, { AxiosInstance } from 'axios';
+
+export default class Api {
```

Comment:
### ❗ Issue: Rename it to `ApiConnection`

The file name is `api.connection` and I like it. To be consistent with file name u need to rename the class name too.